org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/models/ScanModel.java

```diff
@@ -175,7 +189,15 @@ public void setBean(ScanBean bean) {
 	public void setPositionIterable(Iterable<IPosition> positionIterator) {
 		this.positionIterable = positionIterator;
 	}
-
+	
+	public List<IScannable<?>> getScannables() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)